ExampleStraightenWorms/README.md

```diff
@@ -0,0 +1 @@
+Once worms are untangled, this pipeline shows how they can be straightened and aligned with a low-resolution worm atlas to extract localized intensity measurements and compare patterns of reporter signals. Included are steps for identifying secondary objects (fluorescent marker signals) and relating these objects to individual worms, enabling count of signals on a per-worm basis.
```

Comment:
"Secondary objects" has a particular meaning in CP that isn't precisely what you mean here.  You could say "identifying a second type of object" instead.